pom.xml

```diff
@@ -641,6 +641,16 @@
                     <argLine>-Duser.timezone=${project.timezone} -Dfile.encoding=${project.build.sourceEncoding} -Duser.language=${project.language} -Duser.region=${project.region}</argLine>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.0.0</version>
+                <configuration>
+                    <configLocation>checkstyle.xml</configLocation>
```

Comment:
Is the convention for `checkstyle.xml` to live at the top-level, or should it live somewhere else (`conf/`? )?

Reply:
That depends on your personal opinion. From the Java projects I know, people just place it at the same level as the `pom.xml`.

Reply:
I like it when I find certain files at the top level, including:

- README.md
- LICENSE.md
- CONTRIBUTING.md
- Dockerfile
- Vagrantfle

and for Java projects, seeing checkstyle.xml at the top level seems nice. 😄